lighthouse-core/gather/gatherers/source-maps.js

```diff
@@ -124,18 +118,31 @@ class SourceMaps extends Gatherer {
   }
 
   /**
-   * @param {LH.Gatherer.PassContext} passContext
-   * @return {Promise<LH.Artifacts['SourceMaps']>}
+   * @param {LH.Gatherer.FRTransitionalContext} context
    */
-  async afterPass(passContext) {
-    const driver = passContext.driver;
+  async startSensitiveInstrumentation(context) {
+    const session = context.driver.defaultSession;
+    session.on('Debugger.scriptParsed', this.onScriptParsed);
+    await session.sendCommand('Debugger.enable');
```

Comment:
long-term WDYT about splitting out the script parsed bit as a dependency of this and JsUsage? (I think we took a similar note when we did JsUsage)